src/detections/utils.rs

```diff
@@ -112,30 +112,16 @@ pub fn read_json_to_value(filename: &str) -> Result<impl Iterator<Item = Value>,
         let errmsg = format!("Cannot open file. [file:{filename}]");
         return Result::Err(errmsg);
     }
-    let mut reader = BufReader::new(f.unwrap());
-    let mut contents = String::default();
-    reader.read_to_string(&mut contents).ok();
-    let json = format!(
-        "[{}}}]",
-        contents
-            .split('}')
-            .filter(|s| !s.trim().is_empty())
-            .join("},")
-    );
-    let all_record_json: Result<Value, Error> = serde_json::from_str(&json);
-    if let Err(err_msg) = all_record_json {
-        return Err(err_msg.to_string());
-    }
-    let array_records = all_record_json.unwrap().as_array().unwrap().clone();
-    let ret = array_records
+    let reader = BufReader::new(f.unwrap());
+    let ret = reader
+        .lines()
```

Comment:
In this case, wouldn't it cause an error if the json file that did the jq command had newlines in it?

Reply:
Thank you for your review 🙇 I forgot this pattern... Yes, if it's not in JSONL format, you'll get an error.

Reply:
I will check the implementation that can support the above pattern!

Reply:
Ok, this comment is resolved.